### PR TITLE
[15.0][IMP] account_due_list: reconcile accounts domain and filter by default payable and receivables accounts

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Change domain on view Payments and due list in order to see all moves with reconcile accounts

image

- Add two new filters:

"Incoming": Shows all moves with debit > 0
image

"Outgoing": Shows all moves with credit > 0

image

@Tecnativa

TT44917
@pedrobaeza

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
Thanks for the change. The only debatable thing is to remove the filter from invoices. I think both are complementary, as you can have AR/AP accounts, but don't come from invoices, so I would let both, or even don't touch that filter, as invoices by definition don't allow non AR/AP accounts.

File: account_due_list/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -1,2 +1,2 @@
 This module adds a pending payments due list. The list contains every
-expected payment that has been generated by invoices. The list is fully filterable.
+expected payment that has been generated by receivable and payable accounts. The list is fully filterable.
```
Comment (pedrobaeza) [inline account_due_list/readme/DESCRIPTION.rst]:
```suggestion
expected payment that has been generated by reconcilable accounts. The list is fully filterable.
```

File: account_due_list/views/payment_view.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <!--****************************** PAYMENTS ***********************************-->
+    <!--******************************
+    PAYMENTS ***********************************-->
```
Comment (pedrobaeza) [inline account_due_list/views/payment_view.xml]: Short it to one line only:

```suggestion
    <!-- PAYMENTS -->
```

File: account_due_list/views/payment_view.xml [hunk 3]
```diff
@@ -231,7 +234,8 @@
         <field name="view_mode">tree,form,pivot,graph</field>
         <field
             name="context"
-        >{'search_default_posted_moves': 1, 'search_default_from_invoices': 1, 'search_default_not_full_reconciled': 1}</field>
+        >{'search_default_posted_moves': 1, 'search_default_from_invoices': 1,
```
Comment (pedrobaeza) [inline account_due_list/views/payment_view.xml]: Weird that pre-commit is doing this split when previously it was not done nor failed...

**Review by pedrobaeza — APPROVED** [review]

**Comment by carolinafernandez-tecnativa** [comment]
ping @carlosdauden

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge minor

Please fw-port it to v16

**Pull request merged**